react-basic-practice/src/themes/index.ts

```diff
@@ -0,0 +1,16 @@
+import { extendTheme } from "@chakra-ui/react";
+import { fontFamily, fontSizes, fontWeights } from "./typography";
+import { colors } from "./colors";
+import { borderRadius } from "./metric";
+
+const theme = extendTheme({
+  fonts: {
+    body: fontFamily,
```

Comment:
Pls move it to `typography.ts`
and how about heading??

Reply:
i done